gnd/src/main/java/com/google/android/gnd/ui/recorddetails/RecordDetailsFragment.java

```diff
@@ -90,21 +96,23 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
   public void onActivityCreated(@Nullable Bundle savedInstanceState) {
     super.onActivityCreated(savedInstanceState);
     setHasOptionsMenu(true);
-    viewModel.getRecord().observe(this, this::onUpdate);
   }
 
   @Override
   public void onStart() {
     super.onStart();
-    RecordDetailsFragmentArgs args = getRecordDetailFragmentArgs();
-    viewModel.loadRecordDetails(args.getProjectId(), args.getFeatureId(), args.getRecordId());
+  }
+
+  private void setToolbarTitle(String title) {
+    toolbar.setTitle(title);
+  }
+
+  private void setToolbarSubtitle(String subtitle) {
+    toolbar.setSubtitle(subtitle);
   }
 
```

Comment:
got it!

Reply:
@gino-m 

So it looks like we may need these after all. I guess the toolbar isn't available at create time? I get a null object reference when trying to call toolbar::setX. Unfortunately, I believe we're supposed to observe in onCreate to avoid redundant calls because of the way the lifecycle works. https://developer.android.com/topic/libraries/architecture/livedata#observe_livedata_objects. 

For now I've just added back the wrapper methods.

Reply:
I see. In that case we'll also need to check toolbar != null in each method in case load operation finished before onCreateView has completed.